### PR TITLE
Apposite 20487 d mod09 lm

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Queries inline.

### CODE REVIEW

File: Instructions/20487D_MOD09_LAB_MANUAL.md [hunk 1]
```diff
@@ -130,8 +130,8 @@
         }
     ```
 
-15. Add **GenerateJwtToken** method to **UserController** class,  method with **email** and **user** parameters:
-16. Create new token, inside **GenerateJwtToken** method paste the following code:
+6. To the **UserController** class, add the **GenerateJwtToken** method with the *email* and *user* parameters:
```
Comment (VinayApposite) [inline Instructions/20487D_MOD09_LAB_MANUAL.md]: Please verify my edit.

File: Instructions/20487D_MOD09_LAB_MANUAL.md [hunk 2]
```diff
@@ -130,8 +130,8 @@
...
+7. To create a new token, inside the **GenerateJwtToken** method, paste the following code:
```
Comment (VinayApposite) [inline Instructions/20487D_MOD09_LAB_MANUAL.md]: Please verify my edit.

File: Instructions/20487D_MOD09_LAB_MANUAL.md [hunk 3]
```diff
@@ -154,54 +154,54 @@
...
-9. Fill the name of the application.
-10. Under Web **App / Web API** select **Yse**
-11. Under **Reply URL** add the **https://jwt.ms** and click **Create**
-12. From the blade select **User flows**, and click **New user flow** and select **Sign-up and sign in**.
-13. Enter a policy Name for your application to reference.
-14. Select Identity providers and check Email signup. Optionally, you can also select social identity providers, if already configured. Click **OK**.
-15. Select Sign-up attributes. Choose attributes you want to collect from the consumer during sign-up. For example, check Country/Region, Display Name, and Postal Code. Click Create.
+5. Enter the values for **Organization name** and **Initial domain name**, and then click **Create**.
+6. After creating the tenant, click **Directory and subscription filter** and select your tenant.
+7. In the top left-hand corner of the Azure portal, select **All services**, and then search for and select **Azure AD B2C**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD09_LAB_MANUAL.md]: Please verify my edit.

File: Instructions/20487D_MOD09_LAB_MANUAL.md [hunk 4]
```diff
@@ -154,54 +154,54 @@
...
+8. From the blade, select **Applications**, and click **Add**.
+9. Enter the name of the application.
+10. Under **Web App / Web API**, select **Yes**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD09_LAB_MANUAL.md]: Please verify my edit.

File: Instructions/20487D_MOD09_LAB_MANUAL.md [hunk 5]
```diff
@@ -154,54 +154,54 @@
...
+12. From the blade, select **User flows**, click **New user flow**, and then select **Sign-up and sign in**.
+13. Enter a policy name for your application to reference.
+14. Select the identity providers and select **Email signup**. Optionally, you can also select social identity providers, if already configured. Click **OK**.
+15. Select **Sign-up** attributes. Choose attributes you want to collect from the consumer during sign-up. For example, select **Country/Region**, **Display Name**, and **Postal Code**. Click **Create**.
 
 #### Task 2: Add Azure B2C Authentication
 
-1. Create a new **ASP.NET Core** project at [Repository Root]\Allfiles\Mod09\LabFiles\Lab2\Starter. And name it **AzureActiveDirectoryB2C**
+1. In **[Repository Root]\Allfiles\Mod09\LabFiles\Lab2\Starter**, create a new **ASP.NET Core** project, and then name it **AzureActiveDirectoryB2C**.
 2. To open the project in Microsoft Visual Studio Code, and open **Startup.cs** file.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD09_LAB_MANUAL.md]: Instruction not clear.

Comment (slavBarkal) [reply Instructions/20487D_MOD09_LAB_MANUAL.md]: i think its ok  the change that you did.

File: Instructions/20487D_MOD09_LAB_MANUAL.md (hunk 4 above)
Comment (VinayApposite) [reply Instructions/20487D_MOD09_LAB_MANUAL.md]: You can change it to "In the top left-hand corner of the Azure portal, select **All services**, search for **Azure AD B2C**, and then from the results, click **Azure AD B2C**."

**Pull request merged**